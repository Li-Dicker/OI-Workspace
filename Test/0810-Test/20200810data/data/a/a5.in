10
198
54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 54311 
4067 43922 16499 7826 40061 7249 45326 16092 22385 23383 11293 16990 29160 49548 23933 35117 31701 48463 44484 3333 20475 41195 42440 38734 31294 4387 48422 37098 18850 37215 25860 38510 22607 22484 1102 25680 7180 13549 23096 32877 6867 37138 25813 13325 33966 8534 23151 42929 19512 1298 40709 19713 47134 2143 17565 46250 43391 18886 33121 5185 9211 28420 6237 19 12073 28896 8327 37564 46969 13789 28188 13790 12056 35825 49355 18426 29338 26976 4906 38734 12322 42689 29315 40295 30885 17369 48031 49440 17154 26587 44192 3611 43872 24284 5481 25235 24739 26299 46286 8670 38940 1442 39441 29190 31955 33948 415 1635 3754 10428 18054 20506 22731 38170 31859 29270 3639 16924 4072 5245 5219 40991 2526 24463 13704 29638 18605 2111 20785 40558 49652 47656 1582 18589 6348 13578 46760 44289 47798 38505 19135 9933 27235 31713 2315 32955 30592 19737 19048 28052 29648 30212 19827 35207 10389 43990 36142 46225 4766 29625 40068 34813 45190 16593 22292 769 35983 32500 14590 40651 24115 75 20143 45164 28188 1678 47814 31860 7794 27585 5902 44922 46692 39213 48077 33736 23140 38743 30008 47528 24514 3332 25536 25298 14456 2421 22675 5597 
200
19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 19979 
36155 45484 14228 42990 34923 8573 8501 34527 33989 20180 17371 12625 40517 34573 43142 47836 30784 1632 43383 21926 19767 13936 42890 36019 17475 29787 14232 34671 30632 4041 22095 10123 19370 47715 32470 46059 37811 8137 28043 19483 16056 46450 23233 13728 37577 6074 15779 46575 42537 3611 30801 5671 32226 11848 15768 45955 37429 20600 35205 32660 45323 37382 25147 40479 1642 13007 28381 27570 27502 30864 17164 8553 46913 12968 11041 41920 23412 37736 36977 46162 38663 6461 38620 48604 11981 10699 19576 19694 11490 48782 3975 22628 21334 8340 27042 18636 45311 44899 16711 34539 17739 9948 8922 39412 38535 5681 46604 31434 16218 43463 11477 5676 2545 30279 41586 25300 4820 48920 5298 19077 27500 21322 35629 49139 13442 22217 42573 1257 20667 42137 13605 14286 39470 42400 2210 49756 41926 34023 28468 16535 33885 23017 3349 42110 16175 8973 31705 17963 36203 43144 22527 46615 38855 12467 2878 17500 26396 34288 30270 34308 35237 36539 11764 29690 36085 1782 9737 22446 9762 19820 25561 6624 2209 27888 35989 47310 16054 25061 39957 2866 1916 10032 28762 26563 19277 40258 11509 17463 27391 6038 6622 21334 32730 26852 44565 1209 32350 10271 44513 40367 
200
10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 10277 
21648 31162 8716 34938 47831 17353 49138 19245 13395 15352 44196 4922 36105 18215 31862 26211 46304 49807 10302 3889 48006 3531 9279 19455 37865 49187 46778 35858 42179 41316 18333 10287 22386 45128 17102 9091 14468 3766 45940 9122 11726 2502 42649 24404 47228 21166 43080 3354 7306 36619 39853 36133 37518 47656 49267 7046 26378 34631 16508 34264 10307 27505 28122 11933 48498 6543 16870 16117 49457 46497 16968 43352 29056 42836 13410 34289 19455 6679 21696 14640 46120 12352 21990 5639 20916 2312 36757 43731 37005 46455 21852 32654 35778 851 15153 37305 12403 2499 443 44110 22131 48826 38094 26267 2629 27594 46870 40985 2806 14956 44665 15433 15141 6863 31233 30897 21297 7366 21787 37657 37343 34529 5594 3997 16515 29116 30494 34862 29664 1134 19572 23303 31451 18547 44514 35642 11461 26277 41324 11923 22487 37921 34379 12822 18210 18965 18609 2420 43185 30938 21600 1759 33962 28351 9124 22605 20494 25267 47285 12390 37863 16074 24610 41790 32634 49645 10401 38227 22449 14591 35741 36281 36449 37820 27094 2873 26378 8165 33472 9046 36260 12892 23811 8303 13111 17071 25225 3515 28408 19482 2757 4235 24283 7850 32841 6338 10527 49910 22653 24214 
199
88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 88279 
27299 26638 34142 23469 39520 42914 39 48359 37613 25415 32122 46479 1880 10266 48391 5339 40266 23122 20474 49807 4390 45909 39265 32024 27731 29820 39023 33331 8173 15866 21306 47224 20066 13010 13155 27492 41572 17725 22839 13663 461 44513 31725 34978 34325 5466 28730 32276 45928 21516 2890 4179 29323 4331 18393 26889 17980 49340 47605 40619 30849 45232 8128 19815 33651 9700 29961 4100 1128 35853 36345 2190 2779 24160 13703 12584 26898 46479 44566 36384 5910 40010 22283 24895 2161 31379 19171 7351 49811 25059 1543 18009 10247 20537 14467 42656 233 42143 27751 43563 34516 33435 27247 14912 45364 19133 22311 37857 23844 27012 33926 30691 48604 36134 47709 23264 2674 21818 46498 47127 32650 42101 20954 6151 44425 9657 46280 21866 28879 18568 46630 45471 22526 46639 13559 23122 14871 31064 5221 25560 13199 7708 18998 11015 13421 29813 10334 16810 24768 43862 43969 7206 16269 23145 49391 1858 6580 10090 14031 33614 12814 22344 36024 48391 11134 42953 8506 13675 29005 614 39452 24950 4740 2627 17460 12431 42868 11347 21615 21660 24745 12627 32018 44073 16279 48830 46247 34733 29238 25597 23681 21600 45242 8582 18804 3614 13899 46123 41150 
196
827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 827 
4346 33262 48096 30282 10996 8543 19101 49252 28703 39100 9821 45862 41567 61 13394 17002 25456 11128 899 41278 44231 35329 46641 27012 30126 10777 27436 44704 46535 24491 34434 1502 20182 32092 12757 28243 49026 40249 22266 44171 31682 42623 43164 49592 6006 17441 16287 5033 40262 29675 22876 11077 39297 21387 8055 30064 44959 23134 33624 32711 286 41295 41183 48466 39901 6380 13609 39410 13109 33385 7807 15748 15362 29816 2712 33828 32449 31906 14142 10626 40169 13619 40380 38335 47478 26398 1510 29518 17577 43942 43265 19597 5198 41315 843 34811 34905 38735 3451 36416 1865 18144 30901 42087 39759 31251 42627 31864 14358 24683 33521 44317 44160 43146 33767 10823 21202 4670 24950 36941 5576 3047 48801 44170 46082 44059 25137 46933 42104 38318 10223 4614 38771 38535 43306 41685 21099 27475 12229 41134 48292 41141 25843 14914 30541 20512 26860 22197 179 38489 43574 31158 21395 28909 248 28026 29541 30055 8580 44339 48436 4968 33606 47984 3508 35749 15733 30226 38051 32105 10866 22887 5667 29740 38019 36071 40342 13487 32784 36762 38536 43210 42907 23735 5396 32444 19699 31692 34205 18812 43966 5493 45627 33733 34306 12598 
200
17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 17862 
40964 16317 16824 23717 30421 11627 47392 38164 39579 5925 4859 29522 38996 20335 38988 46167 30725 40198 34571 38920 19529 19109 33366 48170 41704 25325 41216 45119 11546 26147 23052 22660 31191 49469 14919 17027 17368 3323 35770 47948 5807 26802 34751 23754 28640 3019 17151 12218 8780 40333 46392 35638 22545 221 39091 12109 38492 46381 44516 14614 8688 6351 31091 11344 27510 29008 5499 20339 18793 29882 28174 26281 20489 16634 44414 48249 22516 16418 26783 35159 6597 9934 38376 26664 38864 23427 41998 27478 16014 34290 18339 45547 25799 4136 19946 12917 13134 731 38724 6719 36520 13336 24719 39982 39944 6740 18108 47761 35924 37019 23871 95 12900 36193 36821 39159 27238 5550 831 42859 1636 25881 49344 33366 16456 4616 16654 13185 10251 7340 5701 16442 16160 40138 20684 2443 20464 12926 27656 48087 34464 25796 28499 23181 8507 45555 23905 46860 30862 36858 3964 28857 10279 8009 42814 1450 17309 33581 24761 21393 41354 2560 9659 38389 20259 40251 24117 13252 21483 18209 36969 1583 17903 42006 38722 36404 49375 48676 6996 14508 10187 12127 19441 6104 48958 1968 13686 18716 33635 38564 2650 37462 44165 13559 12057 11133 24080 15662 17877 23386 
197
71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 71938 
484 18577 8395 4170 18018 2505 6809 31895 13218 8654 42136 33220 33421 46606 8587 15429 9546 4054 13462 7074 3052 28938 15118 17643 4893 9798 20612 13464 41854 11723 37113 37091 45471 4621 34362 41080 31110 7462 25061 35690 17260 6526 13862 40591 21494 34810 8156 825 8080 33090 49433 11257 24537 22699 9185 28089 1029 40432 19041 22617 37667 1010 47569 48540 28622 6714 7689 36171 6032 8899 23183 25553 11750 16572 5677 26515 20446 40284 14301 15233 24530 2821 45781 32622 31417 41732 44794 19117 49090 39638 7466 1351 26513 32223 395 30747 30824 17302 2378 5687 6813 37199 42786 8776 11370 7435 31897 9964 3549 21304 13761 9045 17011 39163 14787 16338 24581 37465 12834 26949 5151 20523 39902 14521 15127 20569 29914 7183 10886 25693 29988 43799 33617 32348 39900 14678 12179 30506 5161 31658 31353 1709 22951 33292 2963 2465 4592 35555 45590 20930 37267 34715 44203 46224 25149 49058 22688 6397 37700 37177 41762 24334 6588 7396 31821 35779 37485 31900 25743 49614 49087 26104 45781 39776 8261 27648 9150 7922 22942 18210 28100 48374 20030 19336 22322 15731 31337 21511 23165 4917 11188 19029 7424 41533 39251 43769 7824 
196
31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 31668 
37046 19387 26080 36847 29910 47495 13766 3958 47906 14318 27331 11615 44441 31783 2205 14583 19332 32018 25557 28512 7425 37005 9605 17072 22455 15955 6709 3199 15667 38655 31963 36865 7358 30185 37187 19806 28095 41066 30825 1620 29001 44790 9196 41637 45353 21122 37759 16802 33045 26293 901 35622 29490 11828 39856 17370 25533 24812 24903 17498 36391 47717 10460 31235 18028 38970 36971 40095 37432 24901 21534 33405 5218 20657 24610 15091 40643 33841 36807 45977 19074 17854 7053 15703 36057 7253 39668 20193 10188 24902 42203 35693 19984 33894 32509 3387 23314 20511 32799 37288 46327 31360 33590 4407 11523 8400 2626 41545 48005 6751 35685 2028 24775 30751 36544 9030 11689 30081 30903 46870 48248 33733 41146 3930 42772 8164 43075 33917 21266 43374 41315 19591 5904 16244 11598 16303 38004 41757 43715 609 41671 12526 28639 44203 13009 46662 14644 40368 47214 2343 29491 39051 33612 32621 42161 41800 24866 34596 38797 34187 30260 21282 7468 37630 18686 8795 37720 21072 21324 3070 39348 19083 37635 4933 3703 37955 23199 30336 48001 27943 17893 30843 27642 32180 20875 37374 18483 33031 44289 45884 33573 23744 12460 7264 47905 44794 
196
68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 68445 
23538 4675 37376 41328 14204 48038 33699 14622 3820 17621 43039 38009 46786 49494 46522 7427 15674 423 24403 48829 48540 13402 32212 41986 46212 12498 39177 13993 4762 48449 9925 31393 32716 13184 23491 38131 19812 27057 22712 29731 45612 14493 12167 1304 38519 36399 17899 28595 23755 24290 25330 1813 30289 9765 4937 30192 20907 8577 5957 31325 15325 5239 34462 35659 28888 8188 30241 30415 46708 5670 43069 12259 45822 30723 35173 2326 29338 13497 40913 11675 13401 9308 41988 46844 31342 9685 38920 21014 35133 11768 33418 7092 7832 17585 20576 45318 30512 35547 36433 7853 23354 39772 41514 8796 34989 21135 45344 33529 22204 12548 49224 1172 2396 5152 24231 22921 1208 11821 6437 46204 10823 15176 13143 45450 15922 19643 29490 32434 43651 11166 42948 14963 14074 19363 13071 36257 21909 45064 11824 29743 12454 1500 39219 1250 16106 5285 45683 15907 47307 38271 15958 45420 43356 31949 13607 22621 1161 9708 23820 19098 49180 4371 31708 32119 17682 22055 34136 22579 47840 30802 31739 9955 38935 13812 32144 43995 19195 40453 26548 2387 47097 48690 11806 415 20477 45892 15677 31758 22025 48521 44440 41214 45229 45186 48782 27250 
196
65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 65402 
8480 41768 10128 379 8130 43515 3701 28200 23564 31305 32118 30839 29299 17277 38478 34215 5271 35583 18234 30460 36767 1105 30383 34629 47033 26620 8343 31507 49933 42700 893 22017 774 38174 7257 48963 30446 45995 16182 357 37916 1617 17977 43753 5006 16011 41546 26106 42469 9341 12263 35057 30847 49120 8352 22913 32794 15886 45230 37602 6392 21752 4023 36584 20196 33018 49379 38902 7899 11736 21700 13256 44795 15282 22609 47883 3514 20009 16090 47361 32790 31 11255 36232 6055 45114 28297 14057 15015 24608 18446 33580 34764 28048 14805 4731 1147 48809 1256 41001 2830 42581 16073 37656 11055 2097 30766 34019 24383 24565 19986 35332 39618 21557 47904 14785 28545 34019 36734 49870 28483 23932 1024 1628 7350 26507 16976 25837 18752 4963 12147 33967 47951 9321 49911 20599 414 10771 2065 11579 18614 48416 30065 11864 18422 40915 26978 48755 13071 49747 3898 9437 30007 11892 8093 23911 11737 41379 34949 47280 30880 22292 31283 4139 40064 1380 15914 3196 23497 32102 8841 24472 34069 21318 1085 28428 9117 22009 48407 36174 952 12393 13889 8033 13675 47219 26801 12264 49878 20352 32110 2623 48490 17094 29637 42645 
