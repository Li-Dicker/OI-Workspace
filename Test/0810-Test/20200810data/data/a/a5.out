Case #1: 14919970
Case #2: 8376820
Case #3: 6333119
Case #4: 21809324
Case #5: 4916838
Case #6: 7901780
Case #7: 18085369
Case #8: 10784170
Case #9: 17724417
Case #10: 16505076
